.github/workflows/harvest-post-raw-epss.yml

```
@@ -0,0 +1,61 @@
+name: Harvest Post Raw EPSS
+
+on:
+  workflow_call:
+    inputs:
+      tag:
+        required: true
+        type: string
+      pack-threads:
+        required: true
+        default: 2
+        type: number
+      pack-windowMemory:
+        required: true
+        default: 4g
+        type: string
+
+jobs:
+  gc:
+    name: Git GC ${{ inputs.tag}}
```

```suggestion
    name: Git GC ${{ inputs.tag }}
```

It's nice for some other similar lines

---

fixed: https://github.com/vulsio/vuls-data-db/compare/d5fb5a356d851ab8f680271dd6d93d5a2ad022fc..dcbef5ab05c1e7d6012ff2f63ee2cae65eb39622